.github/workflows/deploy.yml

```
@@ -11,67 +11,90 @@
+          echo "\$webhookSecret         = \"${{ secrets.WEBHOOK_SECRET }}\";" >> webhook.secrets.php
+          echo "\$webhookUrl            = \"${{ secrets.WEBHOOK_URL }}\";" >> webhook.secrets.php
+          echo "\$webhooksProcessingUrl = \"${{ secrets.WEBHOOK_PROCESSING_HEALTHCHECK_URL }}\";" >> webhook.secrets.php
+          echo "\$webhooksServiceUrl    = \"${{ secrets.WEBHOOK_SERVICE_HEALTHCHECK_URL }}\";" >> webhook.secrets.php
+
+      - name: 📝 Create Version File for API
         run: |
-          echo "<?php" > src/github.secrets.php
-          echo "\$gitHubClientId      = \"${{ secrets.GH_CLIENT_ID }}\";" >> src/github.secrets.php
-          echo "\$gitHubClientSecret  = \"${{ secrets.GH_CLIENT_SECRET }}\";" >> src/github.secrets.php
-          echo "\$gitHubRedirectUri   = \"${{ secrets.GH_REDIRECT_URI }}\";" >> src/github.secrets.php
+          cd src       
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)